src/bun.js/bindings/Constants.h

```diff
@@ -0,0 +1,5 @@
+#include "root.h"
+#include "JavaScriptCore/JSCJSValue.h"
+#include "ZigGlobalObject.h"
+
+JSC::JSValue ZlibConstants(Zig::GlobalObject* globalObject);
```

Comment:
should this just be in process binding constants?

https://github.com/oven-sh/bun/blob/fc6f8937c00cf4ab55a0cea408770f7fea5d64b3/src/bun.js/bindings/ProcessBindingConstants.cpp#L977-L1095